drupal_symfony_inject.install

```diff
@@ -5,6 +5,16 @@
  */
 
 define('SYMFONY_CLASS_LOADER_PROXIES_DIR', 'symfony_class_loader_proxies_dir');
+
+/**
+ * Implements hook_enable().
+ */
+function drupal_symfony_inject_enable() {
+  // Set a low module weight to ensure this module is enabled before those that
+  // depend on it.
+  db_query("UPDATE {system} SET weight = -99 WHERE name = 'drupal_symfony_inject' AND type = 'module' AND weight = 0");
```

Comment:
I think we don't need this as hook_update_N in  https://github.com/Capgemini/drupal_symfony_inject/pull/8 would cover this case as well.